app/posts/manage-school-placements/2024-02-23-beta-location-analysis-phase-3.md

```diff
@@ -0,0 +1,161 @@
+
+Upon investigating the potential of these areas to improve Birmingham's representativeness, we chose Sandwell as the most appropriate region. Staffordshire placement schools had similar characteristics to Birmingham.
+
+![Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Birmingham. The lighter the colour, the greater the number of placement schools.](birmingham-based-providers-expanded-network.png "Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Birmingham. The lighter the colour, the greater the number of placement schools.")
+
+| Local authority | Number of providers in the area | Number of placements schools in the area |
+| --- | --- | --- |
+| Birmingham | 12 | 333 |
+| Staffordshire CC | 9 | 78 |
+| Sandwell | 7 | 77 |
+| Solihul | 8 | 55 |
+| Dudley | 4 | 55 |
+| Warwickshire CC | 8 | 52 |
+| Worcestershire CC | 6 | 47 |
+| Walsall | 7 | 44 |
+| Wolverhampton | 6 | 41 |
+
+### Birmingham and Sandwell
+
+Including both Birmingham and Sandwell in the sample area does not improve the representativeness of any characteristics compared to Birmingham alone. Birmingham's large placement school population is likely a critical factor in this result, as it will require a sizeable contrasting placement school population to improve the representativeness meaningfully. If we choose Birmingham, we should consider multiple areas for expansion.
+
+| Characteristic | Representative across all categories | Representative across categories of interest |
+| --- | --- | --- |
+| Pupil numbers | No | Not applicable |
+| Urban-rural setting | No | No |
+| Percentage of free school meals | No | Not applicable |
+| Establishment type | Yes | Yes |
+| Phase of education | No | Yes |
+| Ofsted rating | No | No |
+| Religious character | No | No |
+
+## Leeds-based providers expanded network
+
+All the providers in Leeds also operate in Bradford, Calderdale, and Kirklees, so these are both excellent options for a second region.
+
+Bradford has many placement schools that are ‘Academies’, so this region has the best potential to improve the representativeness of the Leeds Placement schools.
+
+![Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Leeds. The lighter the colour, the greater the number of placement schools.](leeds-based-providers-expanded-network.png "Heatmap showing the number of placement schools in each local authority that hosted trainees from a provider located in Leeds. The lighter the colour, the greater the number of placement schools.")
+
+| Local authority | Number of providers in the area | Number of placements schools in the area |
```

Comment:
Same as previous comment

Reply:
We decided not to change the heading as it relates to the section.